program-overview.md

```diff
@@ -0,0 +1,33 @@
+### OVERVIEW
```

Comment:
```suggestion
### Vision :rocket: 
```
I think maybe also rename the file to program-vision, some elements in here are covered in the README but I think it would be good to show the direction we want to go with the program

I would also add a small disclaimer that this highlights where we want to take the program in the future and that not all elements will be available in the current iterations of the program (so they don't read too into the timeline)